cargo/flatpak-cargo-generator.py

```diff
@@ -103,7 +103,18 @@ def fetch_git_repo(git_url, commit):
 async def get_git_repo_packages(git_url, commit):
     logging.info('Loading packages from %s', git_url)
     git_repo_dir = fetch_git_repo(git_url, commit)
-    root_toml = load_toml(os.path.join(git_repo_dir, 'Cargo.toml'))
+    if os.path.isfile(os.path.join(git_repo_dir, 'Cargo.toml')):
+        root_toml = load_toml(os.path.join(git_repo_dir, 'Cargo.toml'))
+        toml_dir = '.'
+    else:
+        for entry in os.scandir(git_repo_dir):
+            if entry.is_dir():
+                toml_path = os.path.join(git_repo_dir, entry.name, 'Cargo.toml')
+                if os.path.isfile(toml_path):
+                    root_toml = load_toml(toml_path)
+                    toml_dir = entry.name
+                    break
```

Comment:
Picking up the first found (mostly random) `Cargo.toml` file doesn't seem like the proper way to go.

And what we do if didn't find a `Cargo.toml`? `root_toml` becomes undefined in this case.

Reply:
The script now raises an exception if multiple or none Corgo.toml are found.